### PR TITLE
Updating and cleaning the GUI Toolkits

### PR DESCRIPTION
**Pull request by thiagojedi**
- Added Xwt
- Added the new QtSharp
- Moving Qyoto and Qt4Net to Dead Toolkits as they are... well... dead

### CODE REVIEW

File: docs/gui/index.md [hunk 1]
```diff
@@ -97,25 +97,38 @@ Cons:
 -   Looks alien on non-Windows platforms.
 -   Code that calls the Win32 API is not portable.
 
-Qyoto
------
+Working in Progress
```
Comment (Therzok) [inline docs/gui/index.md]: ~~Working~~ Work in Progress

File: docs/gui/index.md [hunk 2]
```diff
@@ -97,25 +97,38 @@ Cons:
...
-The Qyoto/Kimono languages bindings allow C# and any other .NET language to be used to write Qt/KDE programs. The bindings are autogenerated directly from the Qt/KDE headers, greatly reducing the maintenance effort. KDE doxygen comments are automatically converted to the xml based C# style of comment format. Substrate is well documented and supported. Qyoto 4.0 got nice [Documentation](http://api.kde.org/qyoto-api/) too.
+Xwt
+---
 
-Platforms: Unix, Windows, OSX
+Homepage: https://github.com/mono/xwt
+
+Xwt is a cross-platform toolkit that works by exposing one unified API across all environments. The API is mapped to native controls on each platform.
+
+It can be used as a standalone framework or it can be embedded into existing applications. The Monodevelop uses this toolkit for simple interfaces that needs to feel native across all the platforms, such as Files Dialogs.
```
Comment (Therzok) [inline docs/gui/index.md]: Wrong, File dialogs are natively written on every platform. I don't know what example would be best to use.

**Comment by thiagojedi** [comment]
Updated with the fixes proposed by @Therzok

File: docs/gui/index.md [hunk 3]
```diff
@@ -97,25 +97,38 @@ Cons:
 -   Looks alien on non-Windows platforms.
 -   Code that calls the Win32 API is not portable.
 
-Qyoto
------
+Work in Progress
+===================
 
-Homepage: [Qyoto](http://techbase.kde.org/Development/Languages/Qyoto)
+Listed below is some Toolkits that are in several stages of active development but didn't have a stable release yet.
```
Comment (akoeplinger) [inline docs/gui/index.md]: **are** some toolkits

File: docs/gui/index.md [hunk 4]
```diff
@@ -97,25 +97,38 @@ Cons:
...
-The Qyoto/Kimono languages bindings allow C# and any other .NET language to be used to write Qt/KDE programs. The bindings are autogenerated directly from the Qt/KDE headers, greatly reducing the maintenance effort. KDE doxygen comments are automatically converted to the xml based C# style of comment format. Substrate is well documented and supported. Qyoto 4.0 got nice [Documentation](http://api.kde.org/qyoto-api/) too.
+Xwt
+---
 
-Platforms: Unix, Windows, OSX
+Homepage: https://github.com/mono/xwt
+
+Xwt is a cross-platform toolkit that works by exposing one unified API across all environments. The API is mapped to native controls on each platform.
+
+It can be used as a standalone framework or it can be embedded into existing applications. The Monodevelop uses this toolkit for simple interfaces that needs to feel native across all the platforms, without the need to write specific code.
```
Comment (akoeplinger) [inline docs/gui/index.md]: ~~The Monodevelop~~, just MonoDevelop
 that needs -> that need
write specific -> write platform specific

File: docs/gui/index.md [hunk 5]
```diff
@@ -97,25 +97,38 @@ Cons:
...
+
+Pros:
+-   Extremely simple API.
+-	Native look and feel on each platform.
+-	Can be embed in existing applications.
```
Comment (akoeplinger) [inline docs/gui/index.md]: embedded

**Comment by akoeplinger** [comment]
Thanks, looks good apart from the few grammar fixes :)

**Comment by thiagojedi** [comment]
Thanks. English is not my native language, so it happens to fail me sometimes.

Fixed. :sweat_smile:

File: docs/gui/index.md [hunk 6]
```diff
@@ -97,25 +97,38 @@ Cons:
 -   Looks alien on non-Windows platforms.
 -   Code that calls the Win32 API is not portable.
 
-Qyoto
------
+Work in Progress
+===================
 
-Homepage: [Qyoto](http://techbase.kde.org/Development/Languages/Qyoto)
+Listed below are some Toolkits that are in several stages of active development but didn't have a stable release yet.
```
Comment (Therzok) [inline docs/gui/index.md]: but ~~didn't~~ don't

File: docs/gui/index.md [hunk 7]
```diff
@@ -97,25 +97,38 @@ Cons:
...
-The Qyoto/Kimono languages bindings allow C# and any other .NET language to be used to write Qt/KDE programs. The bindings are autogenerated directly from the Qt/KDE headers, greatly reducing the maintenance effort. KDE doxygen comments are automatically converted to the xml based C# style of comment format. Substrate is well documented and supported. Qyoto 4.0 got nice [Documentation](http://api.kde.org/qyoto-api/) too.
+Xwt
+---
 
-Platforms: Unix, Windows, OSX
+Homepage: https://github.com/mono/xwt
+
+Xwt is a cross-platform toolkit that works by exposing one unified API across all environments. The API is mapped to native controls on each platform.
+
+It can be used as a standalone framework or it can be embedded into existing applications. Monodevelop uses this toolkit for simple interfaces that need to feel native across all the platforms, without the need to write platform specific code.
```
Comment (Therzok) [inline docs/gui/index.md]: ~~Monodevelop~~ MonoDevelop

File: docs/gui/index.md [hunk 8]
```diff
@@ -97,25 +97,38 @@ Cons:
...
-On Debian/Ubuntu, it can be installed by installing the 'qyoto-dev' package.
+QtSharp
+-------
 
-Qt4Dotnet
----------
+Homepage: https://github.com/ddobrev/QtSharp
 
-Homepage: [Qt4Dotnet](http://code.google.com/p/qt4dotnet/)
+QtSharp started as a sucessor of the Qyoto project. It aims to create libraries that wrap [Qt](https://qt-project.org/) thus enabling its usage through C#. It relies on [CppSharp](https://github.com/mono/CppSharp) and generates the wrappers based on a pre-installed Qt set-up.
```
Comment (Therzok) [inline docs/gui/index.md]: ~~set-up~~ setup

File: docs/tools+libraries/dead-toolkits.md [hunk 9]
```diff
@@ -85,4 +83,20 @@ Gaia Ajax Widgets
 
 Gaia AjaxWidgets provides Mono support.
 
+Qyoto
+-----
+
+Homepage: [Qyoto](http://techbase.kde.org/Development/Languages/Qyoto)
+
+The Qyoto/Kimono languages bindings allow C# and any other .NET language to be used to write Qt/KDE programs. The bindings are autogenerated directly from the Qt/KDE headers, greatly reducing the maintenance effort. KDE doxygen comments are automatically converted to the xml based C# style of comment format. Substrate is well documented and supported. Qyoto 4.0 got nice [Documentation](http://api.kde.org/qyoto-api/) too.
```
Comment (Therzok) [inline docs/tools+libraries/dead-toolkits.md]: ~~Qyoto 4.0 got nice [Documentation](http://api.kde.org/qyoto-api/) too.~~

Qyoto 4.0 documentation can be found [here](http://api.kde.org/qyoto-api/).

File: docs/tools+libraries/dead-toolkits.md [hunk 10]
```diff
@@ -85,4 +83,20 @@ Gaia Ajax Widgets
...
+
+Platforms: Unix, Windows, OSX
+
+On Debian/Ubuntu, it can be installed by installing the 'qyoto-dev' package.
```
Comment (Therzok) [inline docs/tools+libraries/dead-toolkits.md]: it can be ~~installed~~ acquired

File: docs/tools+libraries/dead-toolkits.md [hunk 11]
```diff
@@ -85,4 +83,20 @@ Gaia Ajax Widgets
...
+
+Qt4Dotnet
+---------
+
+Homepage: [Qt4Dotnet](http://code.google.com/p/qt4dotnet/)
 
+This is a port of the QtJambi java bindings to .net using IKVM.
```
Comment (Therzok) [inline docs/tools+libraries/dead-toolkits.md]: ~~.net~~ .NET

**Comment by Therzok** [comment]
Sorry I didn't do the remarks earlier, didn't have the time to read through it all thoroughly.

File: docs/gui/index.md (hunk 8 above)
Comment (thiagojedi) [inline docs/gui/index.md]: Just to explain this one: "set up" is how it is on the QtSharp readme.

**Comment by thiagojedi** [comment]
Thank you.

The errors in the Dead Toolkits page were there before my changes. Sorry I didn't see them too.

**Comment by akoeplinger** [comment]
No worries, thanks for contributing!

**Comment by Therzok** [comment]
:heart:

**Pull request merged**